app/views/categories/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+
+<div class="main-title-contaienr">
+  <h2 class="main-title">Categories</h2>
+  <%= button_to "Log out", destroy_user_session_path, method: :delete, class: "btn-log-out" %>
+</div>
+
+<div class="categories">
+  <% @categories.each do |category| %>
+
+    <a href="<%= category_transactions_path(category) %>" class="category-card">
+      <div class="container-category">
+
+        <div class="container-img">
+          <img class="category-image" src="<%= asset_path('transaction.png') %>" alt="<%= category.name %>_image">
+        </div>
+
+        <div class="container-p">
+        <p class="category-name"><%= category.name %></p>
+        <p class="category-amount">$<%= category.total_amount %></p>
+        </div>
+
+      </div>
+    </a>
+  <% end %>
+</div>
+
+<div class="add-category-container">
+<a class="add-category" href="<%= new_category_path %>">
+  <button class="btn-add-category">Add a New Category</button>
+</a>
+</div>

```

Comment:
- [ ] Please add the correct placement of the `Add a New Category` button make it at the bottom of the page and give it a green color like the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/629dedce-3d2d-40a3-a596-edcf63d0691d)  | ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/8de843c0-bcf2-41ce-b95a-a71cb8223cef)  |

Reply:
Done! ✅